docs/api-reference/Bucket.md

```diff
@@ -127,8 +127,8 @@ __Arguments__
 
 __Returns__
 
-```list```: Return value structure has not yet been permanently decided
-Currently, it's a ``tuple`` of ``list`` objects, containing the
+``list``: Return value structure has not yet been permanently decided
+Currently, it's a `tuple` of `list` objects, containing the
 `following`: (directory info, file/object info, delete markers).
```

Comment:
No need to scare people. If the return type changes, the API semver should reflect that.

```suggestion
``list``: Returns a `tuple` of `list` objects formatted as follows: (directory info, file/object info, delete markers)
```

Reply:
Outdated comment - this is no longer in the PR